### ISSUE TITLE
Companion Pets: Ability/Aura Ids

### ISSUE THREAD

**Issue by Studio60** [silverwind/idTip#7]
Could you add the IDs for companion pet abilities and auras (buffs/debuffs/weather during pet battles) to the addon?

**Comment by silverwind** [comment]
Abilities done! These tooltips are quite different to regular tooltips, so the text position is a compromise here:

![abil](https://cloud.githubusercontent.com/assets/115237/5842429/049a6e8e-a1a5-11e4-8944-d52a26df3572.png)

**Comment by silverwind** [comment]
And auras done too. Didn't see any weather on my test battle - if it's missing, please let me know how to get into a battle with weather :)

![aura](https://cloud.githubusercontent.com/assets/115237/5842633/b06b4e8a-a1a6-11e4-8289-02c746cc1539.png)

**Comment by Studio60** [comment]
That was fast. Wow. Great job.

Weather is the result of pet abilities like "Call Lightning" "Call Darkness" "Moonfire", "Sunlight", "Mudslide" etc. and is then displayed in the top center of the screen. I'll check tomorrow if weather tooltips now have Ids,

**Comment by Studio60** [comment]
Weather Effects are not yet displayed with an ID. I also noticed that pet spells in the pet journal were not displaying their respective ID, while the pet battle bar inside pet battles as well as the auras do

**Comment by silverwind** [comment]
@Studio60: Yeah it's not as easy as with spells where a single shared function exists. I'll have to dig deeper.
